eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/Fallback.js

```diff
@@ -115,9 +108,9 @@ export const Fallback = ({
                         })}
                   </Select>
                   {errorCode ? (
-                    <ValidationWarning icon={ValidationErrorIcon}>
+                    <ValidationError>
                       <>{FALLBACK_ERRORS[errorCode]}</>
```

Comment:
I know you didn't do this, but this looks a bit weird to me. Could it be refactored down to this instead?
```suggestion
                     {FALLBACK_ERRORS[errorCode]}
```